### PR TITLE
Update/css  removing sass references

### PR DESCRIPTION
**Pull request by tlovett1**
@timwright12 opening this for you so I can leave comments.

### CODE REVIEW

File: _includes/markdown/CSS.md [hunk 1]
```diff
@@ -323,7 +280,7 @@ Articles worth reading:
 
 <h2 id="responsive-websites">Responsive websites {% include Util/top %}</h2>
 
-We build our websites mobile first. We do not rely on `respond.js` as it does not work well in certain environments. Instead, we leverage the benefits of Sass to manipulate our breakpoints.
+We build our websites mobile first. We do not rely on JavaScript libraries such as `respond.js` as it does not work well in certain environments. Instead, we leverage a natural, movile-first build process and allow sites gracefully degrade.
```
Comment (tlovett1) [inline _includes/markdown/CSS.md:283]: Typo in *movile*

Comment (tlovett1) [reply _includes/markdown/CSS.md:283]: Also, thoughts on touching on small screen first vs. mobile first here?

File: _includes/markdown/CSS.md [hunk 2]
```diff
@@ -337,45 +294,11 @@ Avoid mixing min-width and max-width media queries.
...
-/// @param {string} $breakpoint - breakpoint name
-/// @group mediaqueries
-@mixin at-least($breakpoint) {
-	$device-width: get-breakpoint-width($breakpoint);
-	@media screen and (min-width: $device-width) {
-		@content;
-	}
-}
-```
+Working with build tools that utilize Sass or PostCSS processing, we can take advantages of reusability and avoid having an unmaintainable number of breakpoints. Using variables and reusable code blocks we can lighten the CSS load and ease maintainability.
```
Comment (tlovett1) [inline _includes/markdown/CSS.md:297]: I feel like this is a bit light considering all that was removed. Thoughts?

Comment (jonathantneal) [reply _includes/markdown/CSS.md:297]: While the previous example _was_ kinda large to take in, could there be basic examples of a small-screen-first approach using either **Sass Variables** or **CSS Custom Media Queries**?

```scss
$m-screen: "min-width: 768px";
$l-screen: "min-width: 1024px";

.some_element {
  padding: 10px;

  @media ($m-screen) {
    padding: 20px;
  }

  @media ($l-screen) {
    padding: 30px;
  }
}
```

```css
@custom-media --m-screen (min-width: 768px);
@custom-media --l-screen (min-width: 1024px);

.some_element {
  padding: 10px;

  @media (--m-screen) {
    padding: 20px;
  }

  @media (--l-screen) {
    padding: 30px;
  }
}
```

Feel free to disregard. I love the overall changes! 😄

Comment (timwright12) [reply _includes/markdown/CSS.md:297]: I wouldn't mind adding a small example here. I would either use pseudocode or have an example for Sass and an example for post-CSS. I'd like to stay away from language that tries to define the tool if at all possible.

File: _includes/markdown/CSS.md (hunk 1 above)
Comment (timwright12) [reply _includes/markdown/CSS.md:283]: @tlovett1 What do you think about either just swapping the language here from "mobile" to "small screen" _or_ swapping the language _and_ addressing mobile vs. small screen in another section of the site?

Comment (tlovett1) [reply _includes/markdown/CSS.md:283]: Small screen works for me!

File: _includes/markdown/CSS.md (hunk 2 above)
Comment (tlovett1) [reply _includes/markdown/CSS.md:297]: I feel like a few CSS Custom Media queries makes sense as it's part of the new CSS spec (right?). I'm also fine with pseudocode. Your call, Tim.

**Pull request merged**